syntax = "proto3";

package bakdata.corporate.v1;

message Corporate {
  string id = 1;
  uint32 rb_id = 2;
  string state = 3;
  string reference_id = 4;
  string event_date = 5;
  string event_type = 6;
  Status status = 7;
  string information = 8;
  string company_name = 9;
  message Address {
    string street = 1;
    string plz = 2;
    string city = 3;
  }
  Address address = 10;
  message Person {
    string first_name = 1;
    string last_name = 2;
    string birthday = 3;
    string city = 4;
  }
  Person person = 11;

}

enum Status {
  STATUS_UNSPECIFIED = 0;
  STATUS_INACTIVE = 1;
  STATUS_ACTIVE = 2;
}
